modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/power_of_attorney_requests_controller.rb

```diff
@@ -24,7 +24,7 @@ def index
           { resolution: :resolving }
         ]
 
-        poa_requests = poa_request_scope.includes(includes).limit(100)
+        poa_requests = filtered_poa_requests.includes(includes).limit(100)
```

Comment:
Let's inline everything into the controller action rather than have a function extracted. This let's us wait until there is a compelling motivation for indirection and so we don't need to first unwind an existing indirection to achieve another.